lib/kafkat/command/reassign.rb

```diff
@@ -10,7 +10,12 @@ def run
         topic_name = ARGV.shift unless ARGV[0] && ARGV[0].start_with?('--')
 
         all_brokers = zookeeper.get_brokers
-        topics = topic_name && zookeeper.get_topics([topic_name])
+
+        topics = nil
+        if topic_name
```

Comment:
A few details:
- in command description, rename `[topic]` into `[topics]`, https://github.com/airbnb/kafkat/blob/6adf922884e24c126d9f6c570239a4108e02d73b/lib/kafkat/command/reassign.rb#L6
- rename `topic_name` into `topic_names`, https://github.com/airbnb/kafkat/blob/6adf922884e24c126d9f6c570239a4108e02d73b/lib/kafkat/command/reassign.rb#L10